### ISSUE TITLE
[Proposal] Нерф скелетов

### ISSUE THREAD

**Issue by Akellazp** [TauCetiStation/TauCetiClassic#8510]
#### Подробное описание проблемы

За копеечный ритуал скелетофикации мы получаем имбу, которой похер на космос, газ, лазер и станы. Скелета можно убить только если отбить ноги или гроин, что, учитывая то что скелеты всегда лутают булетпруф, невероятно сложно.

**Comment by Deahaka** [comment]
Скелет всё ещё несложно убиваем в мили. Падает за три удара чем-нибудь сильным и за 5-6 слабеньким. Без булетпруфа умирает с двух выстрелов летала. Также есть универсальное оружие против всех, по типу гранатомёта, флешки, флешера и всего остального. Эта "имба" не требует нерфа, а я бы даже бафнул на имун к подскальзыванию

**Comment by kalazus** [comment]
Гранатамёт 1, флешки контрятся шмотками сб, что они берут с буллетпруфом. Как идти в ближку если они используют лазерки?

**Comment by Deahaka** [comment]
> Гранатамёт 1, флешки контрятся шмотками сб, что они берут с буллетпруфом. Как идти в ближку если они используют лазерки?

Брать абелятив в карго и идти... Брать ЭМИ и идти... Нужно ли учить кого-то ПГшить? Смазка вообще лучшее средство

**Comment by simb11** [comment]
Почему бы не заменить полный резист к ожогам на просто очень высокий?Чтобы они получали только 30-40 процентов берна,но при нанесении скелету 50 урона его конечность сгорала.

**Comment by Deahaka** [comment]
> Почему бы не заменить полный резист к ожогам на просто очень высокий?Чтобы они получали только 30-40 процентов берна,но при нанесении скелету 50 урона его конечность сгорала.

Ты предлагаешь сделать сжигание конечности за 3-4 выстрела. Это тупо конец скелетам. Никто никогда не будет это брать, легче просто удалить уже, а не издеваться.

**Comment by simb11** [comment]
Да чё ж сразу смерть,30-40 процентов это около 15 урона,то есть нужно примерно 4 попадания в ОДНУ конечность для её уничтожения.Да и камон,у скелетов и так миллион всяких фич,они могут спокойно делать рцд в пол и убивать всех разгермой,они могут легко убивать всех при помощи дубинок и бол,они могут в любой момент уйти к себе в рай и так далее.

**Comment by ramiro181** [comment]
Если я что-то понимаю в коде:

Буллетпруф броник = 80% резиста к пулям.

	if(brute < BP.min_broken_damage)
		lose_bone_chance = 5
	else if(brute < BP.min_broken_damage * 2)
		lose_bone_chance = 20
Че пацаны, шансы.

/obj/item/organ/external/l_arm/skeleton
min_broken_damage = 10

/obj/item/organ/external/r_leg/skeleton
min_broken_damage = 10

Для 20% шанса дропа конечности - 10 дамага. В буллетпруфе - 50 брута. Для гарантированного - 20. В буллетпруфе - 100. Причем не накопленного, а разового, насколько я понял.
Самый большой дамаг у нас у ПТР - 110. Слаг - 60. АК74 - 35. Стандартный глоковский летал - 25. Пули из AC-2 Ultra - 20.
По итогу получаем, что скелеты в бронике кушают пули пачками с _пятипроцентным_ шансом лишиться конечности, которую можно тут же прикрутить обратно.

А еще там есть такие микроприколы:
	
if(prob(lose_bone_chance))
		if(!BP.cannot_amputate)
			BP.droplimb(null, null, DROPLIMB_EDGE)
		else if(BP.children.len) // **hitting the chest will drop random attached bone**
			var/obj/item/organ/external/OBP = pick(BP.children)
			OBP.droplimb(null, null, DROPLIMB_EDGE)

**Comment by Deahaka** [comment]
> Да чё ж сразу смерть,30-40 процентов это около 15 урона,то есть нужно примерно 4-5 попаданий в ОДНУ конечность для её уничтожения.Да и камон,у скелетов и так миллион всяких фич,они могут спокойно делать рцд в пол и убивать всех разгермой,они могут легко убивать всех при помощи дубинок и бол,они могут в любой момент уйти к себе в рай и так далее.

Любой может уйти к себе в рай, но как мы знаем это не всегда успевается. Сжигание гроина = смерть, с 12 урона (70% резиста бёрна) отлетит с четырёх выстрелов, с 16 (60% резиста к бёрну) отлетит за 3. РЦД в пол это не стандартная тема, учитывая что скелеты падают от ветерка. И вообще достать РЦД достать тоже не каждый раунд можно. Дубинка и болы это как раз тактика против скелета, а не за него.

**Comment by Deahaka** [comment]
> > Ты предлагаешь сделать сжигание конечности за 3-4 выстрела. Это тупо конец скелетам. Никто никогда не будет это брать, легче просто удалить уже, а не издеваться.
> 
> Эх, учи мат часть, 70% резиста и 50 хп это 166,(6) урона для уничтожения, 8+ выстрелов, если брать стандартные 20 урона. И это без брони.

У конечности 50 хп. Выстрел лазера - 40 урона... 0% брони = 40 урона, 50% брони = 20 урона. 70% брони = 16 урона. 50/16=4. Или это я придумал математику липовую, чтобы обсчитывать и ограблять шизов? Ещё стоит учесть что большая часть брони на станции хуже всего защищает от лазера, а не от баллистики.

**Comment by kalazus** [comment]
> У конечности 50 хп. Выстрел лазера - 40 урона... 0% брони = 40 урона, 50% брони = 20 урона. 70% брони = 16 урона. 50/16=4. Или это я придумал математику липовую, чтобы обсчитывать и ограблять шизов? Ещё стоит учесть что большая часть брони на станции хуже всего защищает от лазера, а не от баллистики.

Да, брал базовый урон 20, забыв что лазерки у нас по 40 наносят, но коэффициенты всегда можно трогать чтобы сбалансировать. Да ту-же регенерацию хп конечности добавить, раз они или полностью рабочие, или отделены от тела.
